core/playwright/externalFormCreatePub.spec.ts

```diff
@@ -20,6 +20,12 @@ let page: Page;
 
 test.beforeAll(async ({ browser }) => {
 	page = await browser.newPage();
+	page.on("console", async (msg) => {
+		if (msg.type() === "error") {
+			// eslint-disable-next-line no-console
+			console.error("Error:", msg, msg.location());
+		}
+	});
```

Comment:
i was having big troubles with the tests, and this shows the browser error logs in the playwright logs.
let me know if i should remove it